gstd/src/lib.rs

```diff
@@ -28,6 +28,25 @@
 //! asynchronous programming primitives, arbitrary types encoding/decoding,
 //! providing convenient instruments for creating programs from programs, etc.
 //!
+//! # Crate features
```

Comment:
For me it's a good practice to doc crate features be placed/duplicated in toml

Reply:
Fixed in 0defe87.